app/scripts/lib/assertion.js

```diff
@@ -5,11 +5,12 @@
 define(function (require, exports, module) {
   'use strict';
 
+  var Duration = require('durationJs');
   var jwcrypto = require('vendor/jwcrypto');
   var P = require('lib/promise');
 
-  var CERT_DURATION_MS = 1000 * 60 * 60 * 6; // 6hrs
-  var ASSERTION_DURATION_MS = 1000 * 3600 * 24 * 365 * 25; // 25 years
+  var CERT_DURATION_MS =  new Duration('6h').milliseconds();
+  var ASSERTION_DURATION_MS = new Duration('52w').milliseconds() * 25; //25 years
```

Comment:
I think if we're talking 25 years (2041), I'm personally not too concerned about it being +/-1 month. That said, possibly not a horrible idea to create a simple date lib which just does the 11 lines of code we need from moment-js. Or just merge as-is, and file a new bug to consider in the future.